README.md

```diff
@@ -43,19 +43,19 @@ ToolJet is an **open-source low-code framework** to build and deploy internal to
 
 
 
-## All features
+## List of features available:
```

Comment:
Title change is not required. Please revert.

Reply:
done!